src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/ja-JP/strings.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": "1.0.0.0",
   "strings": {
-    "name": "Blazor サーバー アプリ",
-    "description": "ASP.NET Core アプリ内のサーバー側で実行され、SignalR 接続を介してユーザー操作を処理する Blazor サーバー アプリを作成するためのプロジェクト テンプレート。このテンプレートは、高度でダイナミックなユーザー インターフェイス (UI) を備えた Web アプリに使用できます。"
+    "name": "Blazor Server アプリ",
```

Comment:
It seems that the word `Server` is not getting translated anymore in various languages. Is it expected?

Reply:
It may be branding or be defined as a term in term studio. Keep in mind these strings have not been refreshed in over a year with new loc rules/terms